### DISCUSSION TITLE
Nport Moxa

### DISCUSSION THREAD

**Discussion by gigich** [cbrosius/luxtronik_v1#25] [General]
Salü
Ich denke du verstehst Deutsch oder?
Ich bin per Zufall auf dein Projekt gestossen. Ich habe eine LWC80 mit Luxtronik 1 und daran seriell eine Moxabox. Mit dem Nport-Treiber lässt sich so per Ip auf die Seriellschnittstelle zugreifen. Aktuell verbinde ich mit der Windows-Software Luxcontrol darauf um manuell Einstellungen vorzunehmen oder die Werte anzuschauen. 

Ich habe weiter einen Raspberry Pi 5 mit Home Assistant OS und mehreren Docker Container drauf. 

Einen Nport-Treiber gibts auch für Linux, weshalb ich Hoffnung habe so auch vom HASSO per Ip auf die serielle Schnittstelle zugreifen zu können. 

Meinst du das könnte klappen und wie könnte ich das angehen? 

Kannst du auch Befehle senden?

Die Werte könnte ich dann in InfluxDB speichern und mit Grafana auswerten. 

Wenn man aber auch steuern könnte wäre es auch möglich die Heizung und Warmwasser dynamisch aufgrund des Stromverbrauchs (PV-Überschuss) zu steuern. 

Danke für dein Feedback

Gruss Lukas

**Comment by cbrosius** [comment]
Prinzipiell kann das funktionieren, ich weiß aber nicht, ob es eine
Linux-Software dafür gibt. Man kann Befehle senden, muss aber vorher in den
Programmiermodus schalten und hinterher den neuen Wert abspeichern. Ob der
interne flash das lange mitmacht kann dir auch keiner sagen. Ich selbst
lese aktuell nur die Temperaturen und Zustände aus und stelle die grafisch
dar.

gigich ***@***.***> schrieb am So., 21. Juli 2024, 16:06:
